packages/create-nuxt-app/lib/saofile.js

```diff
@@ -125,6 +125,27 @@ module.exports = {
       }
     })
 
+    if (this.answers.language.includes('ts')) {
+      // Add types for any additional selected package
+      const hasJest = this.answers.test.includes('jest')
+      const hasAxios = this.answers.features.includes('axios')
+      actions.push({
+        type: 'modify',
+        files: 'tsconfig.json',
+        handler (data) {
+          const { compilerOptions } = data
+          const { types } = compilerOptions
+          if (hasJest) {
+            types.push('@types/jest')
```

Comment:
Should we add `@types/jest` and `@nuxtjs/axios` in corresponding package.json  ?

Reply:
They are actually already there as a sub-dependency, but i'll add them to package.json to make it more explicit.

Reply:
I've added the Jest-types to `` package.json```

```@nuxtjs/axios```is already part of ```package.json``` when axios is selected, so no need for that one.